cmd/cluster-network-operator/main.go

```diff
@@ -108,8 +113,62 @@ func main() {
 
 	log.Print("Starting the Cmd.")
 
-	// Start the Cmd
-	if err := mgr.Start(signals.SetupSignalHandler()); err != nil {
+	// Create the stop channel and start the operator
+	stop := signals.SetupSignalHandler()
+	if err := start(mgr, stop); err != nil {
 		log.Fatal(err)
 	}
 }
+
+// start creates the default Proxy if it does not exist and then
```

Comment:
If the default proxy is null, does it make sense to just add it to `manifests` and have the CVO create it?

Reply:
or the cluster-config-operator.

Reply:
> If the default proxy is null, does it make sense to just add it to manifests and have the CVO create it?

It's my understanding that https://github.com/openshift/cluster-config-operator/pull/72 does this. This chunk of code tries to solve the problem of "what if someone deletes the cluster proxy". It's my understanding that this object should always exist, but nothing prevents a user from deleting the resource.

Reply:
@squeed I'm going to remove the code from `main.go` since I discovered that the proxy "cluster can not be deleted from the apiserver:
`Error from server (Forbidden): proxies.config.openshift.io "cluster" is forbidden: deleting required proxies.config.openshift.io resource, named cluster, is not allowed`